apps/condo/bin/delete-meter-readings-by-property-ids.js

```diff
@@ -0,0 +1,79 @@
+const path = require('path')
+const { find } = require('@core/keystone/schema')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const get = require('lodash/get')
+const { MeterReading, Meter } = require('@condo/domains/meter/utils/serverSchema')
+
+class FixMeterReadingsClients {
+    context = null
+    meters = []
+    meterReadings = []
+
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
+        const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+        await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+        await keystone.connect()
+        this.context = await keystone.createContext({ skipAccessControl: true })
+    }
+
+    async findMeterReadings () {
+        this.meterReadings = await find('MeterReading', {
+            meter: { property: { id_in: ['3377a303-b098-426a-b559-e0acdf57071e', '19e7b8f0-d204-4f1e-80df-6069f01b15b4'] } },
```

Comment:
May be pass property id as script args? And throw error is not passed?

Reply:
propertyIds variable and throwing of exception added